src/RoleObjects/RadioData.js

```diff
@@ -15,6 +16,8 @@ export default class RadioData extends InputTagData {
    * Sets the value of the radio button
    * @access private
    */
+
+
```

Comment:
Nit: please delete these blank lines.  A function comment should be right next to its associated function.

Reply:
Done